packages/shell-api/src/abstract-cursor.ts

```diff
@@ -83,7 +83,6 @@ export abstract class AbstractCursor<CursorType extends ServiceProviderAggregati
   async* [Symbol.asyncIterator]() {
     let doc;
     // !== null should suffice, but some stubs in our tests return 'undefined'
-    // eslint-disable-next-line eqeqeq
```

Comment:
I'm wondering if we shouldn't actually add this rule to the shared config. Surprised to find that it isn't there.

Reply:
Yeah, me too

Reply:
https://jira.mongodb.org/browse/MONGOSH-1505